### PR TITLE
Portfolio website: Work on making page responsive on all devices

### PR DESCRIPTION
**Pull request by TaiwoEnoch**
In this **Pull Request**, I worked on the following key pivots for my portfolio:

- [x] Use media queries in your CSS file to make your desktop version look like the template you selected (Desktop: 768px or wider).
- [x]  In order to lay out the elements on the page you should use Grid and Flexbox.
You need to make sure that your dekstop version looks like the one in the Figma design template that you chose. We include a screenshot of Template 1 below for reference, but you should use the template that you chose.
- [x] Ignored the part of the design template and use only one input for the full name for all versions.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -170,17 +188,23 @@ body {
 }
 
 .work-showcase {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
   background: #fff;
   padding-bottom: 120px;
+  width: 100%;
+  margin: 0 auto;
```
Comment (TadesseDev) [inline styles.css:197]: - [Optional] Your design is really interesting @TaiwoEnoch, Just a suggestion here is that your project list is a bit darker compared with the Figma design. It's a major skill to be able to translate a design into its exact replica, hence You can work on the overlay of a dark shadow on your projects to make it look like the Figma design.

### Your design 
Screenshot 2023-01-21 011050

### Figma design 
Screenshot 2023-01-21 011114

**Review by TadesseDev — APPROVED** [review]
### Hi @TaiwoEnoch 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)


## Highlights
 - [ ]  Correct GitHub Flow. ✅
 - [ ]  Good commit history. ✅
 - [ ]  Good PR title/description. ✅
 - [ ]  All linters passed ✅
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (TaiwoEnoch) [reply styles.css:197]: Thank you for your observation I'm grateful. 👍

**Comment by TaiwoEnoch** [comment]
Finished my desktop version of my portfolio for responsiveness

**Pull request merged**